golem_messages/datastructures.py

```diff
@@ -68,13 +68,35 @@ def __setitem__(self, key, value):
         validator = getattr(
             self, 'validate_{}'.format(key), None
         )
-
         if callable(validator):
             validator(value=value)  # noqa pylint:disable=not-callable
 
         super().__setitem__(key, value)
 
 
+class ValidatingDependentDictItems(ValidatingDict):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.validate_dependent_dict_items()
+
+    def validate_dependent_dict_items(self):
+        dependent_items_list = getattr(self, 'DEPENDENT_ITEMS', None)
+        for items in dependent_items_list:  # pylint: disable=not-an-iterable
+            validator_function_dict = dict()
+            assert len(items) == 2
+            validator_function_name = "validate_" + "_".join(items)
```

Comment:
Use `f''` here if number of `items` is known and low.

Reply:
Yes it work. I create this class to validate fields which are connected. Also validation works. There are tests for it.

Reply:
>Use f'' here if number of items is known and low. 

Done